packages/twenty-server/src/engine/core-modules/workflows/commands/start-workflow.command.ts

```diff
@@ -0,0 +1,62 @@
+import { Logger } from '@nestjs/common';
+
+import { Command, CommandRunner, Option } from 'nest-commander';
+
+import { InjectMessageQueue } from 'src/engine/integrations/message-queue/decorators/message-queue.decorator';
+import { MessageQueue } from 'src/engine/integrations/message-queue/message-queue.constants';
+import { MessageQueueService } from 'src/engine/integrations/message-queue/services/message-queue.service';
+
+interface RunCommandOptions {
+  workspaceId?: string;
+}
+
+@Command({
+  name: 'workflows:start',
+  description: 'Start a workflow manually',
+})
+export class StartWorkflowCommand extends CommandRunner {
+  private readonly logger = new Logger(StartWorkflowCommand.name);
+
+  constructor(
+    @InjectMessageQueue(MessageQueue.cronQueue)
+    private readonly messageQueueService: MessageQueueService,
+  ) {
+    super();
+  }
+
+  async run(_passedParam: string[], options: RunCommandOptions): Promise<void> {
+    this.logger.log('Hey! Command started');
+
+    this.messageQueueService.addFlow({
+      name: 'renovate-interior',
```

Comment:
Could we find a common signature with the current way we add jobs? The only difference for flows is to be able to set Children jobs I believe